app/controllers/middleware_server_controller.rb

```diff
@@ -9,13 +9,37 @@ class MiddlewareServerController < ApplicationController
   after_action :set_session_data
 
   OPERATIONS = {
-    :middleware_server_reload => {:op   => :reload_middleware_server,
-                                  :hawk => N_('Not reloading Hawkular server'),
-                                  :msg  => N_('Reload initiated for selected server(s)')
+    :middleware_server_reload   => {:op   => :reload_middleware_server,
+                                    :skip => true,
+                                    :hawk => N_('reloading'),
+                                    :msg  => N_('Reload')
+  },
+    :middleware_server_stop     => {:op   => :stop_middleware_server,
+                                    :skip => true,
+                                    :hawk => N_('stopping'),
+                                    :msg  => N_('Stop')
     },
-    :middleware_server_stop   => {:op   => :stop_middleware_server,
-                                  :hawk => N_('Not stopping Hawkular server'),
-                                  :msg  => N_('Stop initiated for selected server(s)')
+    :middleware_server_restart  => {:op   => :restart_middleware_server,
+                                    :skip => true,
+                                    :hawk => N_('restarting'),
+                                    :msg  => N_('Restart')
+    },
+    :middleware_server_shutdown => {:op    => :shutdown_middleware_server,
+                                    :skip  => true,
+                                    :hawk  => N_('shutting down'),
+                                    :msg   => N_('Shutdown'),
+                                    :param => ':timeout'
```

Comment:
Good catch. That is fallout from the first implementation in #8210 and my trial to implement it differently